### PR TITLE
[CHECKOUTUI-40] Inconsistency in the delivery field

### PR DESCRIPTION
**Pull request by Arturo777**
https://vtex-dev.atlassian.net/browse/CHECKOUTUI-40

The brazilian partner bube is working with the app and noticed that apparently in the Checkout Ui Custom sometimes an inconsistency in relation to the delivery field located in the cart happens, sometimes it appears and sometimes not. I did some tests and the and they had similar results. In this cart (https://bube.myvtex.com/checkout/cart/add/?sku=5380&qty=1&seller=1&sc=1) the field only appears when i try to change the quantity of products. 

Workspace to test: https://vtexapps--bube.myvtex.com

### CODE REVIEW

**Review by albertm805 — DISMISSED** [review]
Can you run `vtex run app` and add the compiled files to this repo please?

File: checkout-ui-custom/src/_js/_v.custom.checkout.ui.js [hunk 1]
```diff
@@ -47,7 +47,11 @@ class checkoutCustom {
 
   builder() {
     const _this = this
-
+    if ($('.cart-more-options').length === 0) {
+      return setTimeout(() => {
+        _this.builder()
```
Comment (lucvysk) [inline checkout-ui-custom/src/_js/_v.custom.checkout.ui.js:52]: This is not good, man! `builder()` is calling `builder()` after 1.5s and `ajaxComplete` calls builder(). There's not even a debounce.

**Review by lucvysk — CHANGES_REQUESTED** [review]
please, try to put it on the calls in `start()`

File: checkout-ui-custom/src/_js/_v.custom.checkout.ui.js (hunk 1 above)
Comment (Arturo777) [reply checkout-ui-custom/src/_js/_v.custom.checkout.ui.js:52]: No problem, the builder function needs `$('.cart-more-options')` existing, `ajaxComplete` executes the `init` function and `init` executes `builder` then builder is calling builder only if `$('.cart-more-options')` exists

**Review by lucvysk — APPROVED** [review]

**Pull request merged**